csharp/Platform.Setters/Setter[TResult, TDecision].cs

```diff
@@ -6,21 +6,49 @@
 namespace Platform.Setters
 {
     /// <summary>
-    /// <para>Represents implementation for an setter that allows you to set a passed value as the result value. This setter implementation has additional methods that, simultaneously with setting the result value, return <typeparamref name="TDecision"/> values indicating true or false.</para>
-    /// <para>Представляет реализацию для установщика, который позволяет установить переданное ему значение в качестве результирующего значения. В этой реализации установщика есть дополнительные методы, которые одновременно с установкой результирующего значения возвращают значения типа <typeparamref name="TDecision"/>, обозначающие истину или ложь.</para>
+    /// <para>Represents a setter that allows to set a passed value as the result. This setter variant has additional methods that, simultaneously with setting the result, return <typeparamref name="TDecision"/> values indicating true or false.</para>
+    /// <para>Представляет реализацию для установщика, который позволяет установить переданное ему значение в качестве результата. В этой реализации установщика есть дополнительные методы, которые одновременно с установкой результата возвращают значения типа <typeparamref name="TDecision"/>, обозначающие истину или ложь.</para>
     /// </summary>
-    /// <typeparam name="TResult"><para>The type of result value.</para><para>Тип результирующего значения.</para></typeparam>
-    /// <typeparam name="TDecision"><para>The type of value which will be used to make the decision.</para><para>Тип значения на основе которого будет приниматься решение.</para></typeparam>
+    /// <typeparam name="TResult">
+    /// <para>The type of a result value.</para>
+    /// <para>Тип результирующего значения.</para>
+    /// </typeparam>
+    /// <typeparam name="TDecision">
+    /// <para>The type of values indicating true and false.</para>
+    /// <para>Тип значений обозначающих истину и ложь.</para>
+    /// </typeparam>
     public class Setter<TResult, TDecision> : SetterBase<TResult>
     {
+        /// </summary>
+        /// <param name="TrueValue">
```

Comment:
```suggestion
        /// <summary>
```
This was a closing tag, here we should use an opening tag for `summary`.